src/SlashCommands.tsx

```diff
@@ -141,13 +141,25 @@ export class Command {
 
     run(roomId: string, threadId: string, args: string) {
         // if it has no runFn then its an ignored/nop command (autocomplete only) e.g `/me`
-        if (!this.runFn) return reject(_t("Command error"));
+        if (!this.runFn) {
+            return reject(
+                newTranslatableError(
+                    "Command error: This command should have been handled by another handler and we" +
+                        "  can't try to run something without a command function.",
+                ),
```

Comment:
Updated error string here so if it happens, there is more info to pass along

Reply:
this seems full of jargon, I know slash commands are power-usery but still this feels a bit much

Reply:
(language feedback welcome)

This is another one of the errors that users shouldn't see and I think we should just report it.

The error before was useless and ambiguous with the other `Command failed` throws to make it harder to pick out which place threw.

Reply:
Perhaps `Unable to handle slash command`